Upstream-Status: Backport
[https://github.com/MusicPlayerDaemon/MPD/commit/f869593ac8913e52c711e974257bd6dc0d5dbf26]

Signed-off-by: Wang Mingyu <wangmy@fujitsu.com>

From f869593ac8913e52c711e974257bd6dc0d5dbf26 Mon Sep 17 00:00:00 2001
From: Max Kellermann <max.kellermann@gmail.com>
Date: Mon, 15 May 2023 20:59:58 +0200
Subject: [PATCH] TimePrint: minor fixup for libfmt 10

libfmt version 10 has difficulties formatting a `StringBuffer`, and we
need to help it by explicitly invoking the `c_str()` method.
---
 src/TimePrint.cxx | 2 +-
 1 files changed, 1 insertions(+), 1 deletion(-)

diff --git a/src/TimePrint.cxx b/src/TimePrint.cxx
index 5bf05f6238..d47f3178bb 100644
--- a/src/TimePrint.cxx
+++ b/src/TimePrint.cxx
@@ -20,5 +20,5 @@ time_print(Response &r, const char *name,
 		return;
 	}
 
-	r.Fmt(FMT_STRING("{}: {}\n"), name, s);
+	r.Fmt(FMT_STRING("{}: {}\n"), name, s.c_str());
 }
